cluster/src/main/java/org/apache/iotdb/cluster/server/member/RaftMember.java

```diff
@@ -439,6 +417,31 @@ public HeartBeatResponse processHeartbeatRequest(HeartBeatRequest request) {
     }
   }
 
+  private void tryUpdateCommitIndex(long leaderTerm, long commitIndex, long commitTerm) {
+    if (term.get() == leaderTerm && logManager.getCommitLogIndex() < commitIndex) {
```

Comment:
```suggestion
    if (leaderTerm >= term.get() && logManager.getCommitLogIndex() < commitIndex) {
```

Reply:
Maybe, we can commit the log as long as the leader's term larger than the follower's term?